server/routes/event.js

```diff
@@ -4,34 +4,42 @@ import validate from 'express-validation'
 import eventController from '../controllers/event'
 import eventValidation from '../validation/event'
 
-import { canReadAsVisitor, canCreate, canUpdate, canDelete } from '../middlewares/roles'
+import { canCreate, canUpdate, canDelete } from '../middlewares/roles'
+import { grantWhenFestivalFree } from '../middlewares/authorizeFreeFestival'
+import { authorizeJWT } from '../middlewares/authorizeJWT'
+import { getUserWhenAuthenticated } from '../middlewares/getUserWhenAuthenticated'
 
 const router = express.Router() // eslint-disable-line new-cap
 
 router.route('/')
   .get(
-    canReadAsVisitor,
+    grantWhenFestivalFree,
+    getUserWhenAuthenticated,
```

Comment:
@sandreae, noticed an issue when testing H3K. This throws an error when I'm logged in and festival is not free. I think the order of the middlewares has to be changed (first getUser then grantWhen). Same for all other routes using this logic.

Reply:
Ah! Thanks, good spot. Will fix it.